src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/Conversions.cs

```diff
@@ -194,10 +194,12 @@ protected override Conversion GetInterpolatedStringConversion(BoundExpression so
                 }
 
                 if (elements.Length > 0 &&
-                    !_binder.HasCollectionExpressionApplicableAddMethod(syntax, targetType, elementType, addMethods: out _, BindingDiagnosticBag.Discarded))
+                    !_binder.HasCollectionExpressionApplicableAddMethod(syntax, targetType, addMethods: out _, BindingDiagnosticBag.Discarded))
                 {
                     return Conversion.NoConversion;
                 }
+
+                // PROTOTYPE: Probably should at least accumulate dependencies and propagate them in case of success.
             }
 
```

Comment:
>// PROTOTYPE: Probably should at least accumulate dependencies and propagate them in case of success. [](http://example.com/codeflow?start=16&length=101)

Logged [https://github.com/dotnet/roslyn/issues/72770](https://github.com/dotnet/roslyn/issues/72770).